internal/wireguard/rule.go

```diff
@@ -13,6 +14,29 @@ func (w *Wireguard) addRule(rulePriority, firewallMark int) (
 	rule.Priority = rulePriority
 	rule.Mark = firewallMark
 	rule.Table = firewallMark
+	rule.Family = unix.AF_INET //nolint:all
```

Comment:
Please specify the linters complaining with i.e. `//nolint:revive,nosnakecase` instead of using `all`